### DISCUSSION TITLE
Is it possible to stop uploading artifacts to the download site if artifatcts with the same vesion exist?

### DISCUSSION THREAD

**Discussion by thagikura** [hydraulic-software/conveyor#144] [Q&A] [answered]
When I run `conveyor make copied-site` with artifacts of the same version already uploaded to the download site, I see a warning like:

```
Version X.X.X.X of your Mac app already exists. Are you rebuilding an existing version?
```

Is there a way to prevent uploading the artifacts to avoid unintentional overwrites?

**Comment by mikehearn** [comment, marked as answer]
Unfortunately there isn't any way to convert warnings into errors, but that's a good idea! I filed a ticket to allow escalation of warnings to errors.

**Reply by thagikura** [reply]
Thanks! Looking forward to the update.
